cfripper/config/config.py

```diff
@@ -68,6 +76,8 @@ class Config:
         "directconnect:",
     ]
 
+    rules_config_schema = Schema({str: RuleConfig})
```

Comment:
What's the difference between schema and pydantic for model validation?

Reply:
In this case we do not need any parsing for this structure, so I assumed a pydantic model class was not needed. In order to do a simple schema validation, this was the simplest way I could think of.